BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -190,3 +192,44 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
     }
     
 }
+
+extension StoresTableViewController {
+    
+    func setupRefreshControl() {
+        
+        refreshControl = UIRefreshControl()
+        refreshLoadingView = UIView(frame: self.refreshControl!.bounds)
```

Comment:
This is a case where forced unwrap is acceptable, though I would still do a `guard let` statement anyway.